.rubocop_todo.yml

```diff
@@ -0,0 +1,7 @@
+# This configuration was generated by
+# `rubocop --auto-gen-config`
+# on 2017-10-02 16:26:07 -0400 using RuboCop version 0.50.0.
+# The point is for the user to remove these configuration records
+# one by one as the offenses are removed from the code base.
+# Note that changes in the inspected code, or installation of new
+# versions of RuboCop, may require this file to be generated again.
```

Comment:
I think this is a left-over and should be removed, no?

Reply:
Yeah, this project is doing too much work for Rubocop. Usually we create a .rubocop.yml with `inherit_from: .rubocop_todo.yml` and run `rubocop -a ; rubocop --auto-gen-config` and do zero thinking about what should go in the main yml to avoid hurting our heads. I'll PR that separately.

Reply:
I totally disagree, sorry! I think a `.rubocop_todo.yml` is an antipattern - why run RuboCop if you're just going to ignore what it says!! 😝